cross/dkb-trace/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = dkb-trace
+PKG_VERS = 2.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://digitalbox.chez.com/sources
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = 
+
+HOMEPAGE = http://digitalbox.chez.com/
```

Comment:
This field is for the project homepage, helpurl. Example: http://aminet.net/package/gfx/3d/DKBTrace

Reply:
Ok, indeed I can add the my GitHub homepage instead